apps/testingaccessibility/src/components/portable-text/index.tsx

```diff
@@ -242,20 +242,32 @@ const PortableTextComponents: PortableTextComponentsProps = {
     },
     bodyImage: ({value}: BodyImageProps) => <BodyImage value={value} />,
     code: ({value}: CodeProps) => {
-      const {language, code, highlightedLines} = value
-
+      const {language, code, highlightedLines, filename: label} = value
       return (
-        <Refractor
-          language={
-            language
-              ? Refractor.hasLanguage(language)
-                ? language
-                : 'javascript'
-              : 'javascript'
-          }
-          value={code}
-          markers={highlightedLines}
-        />
+        <>
+          <pre
+            role="region"
+            aria-label={label ? label : 'code sample'}
+            tabIndex={0}
+            className="sr-only"
+          >
+            <code>{code}</code>
+          </pre>
+          <pre aria-hidden="true">
+            <Refractor
```

Comment:
@marcysutton I noticed that screen readers read the whole code as a single sentence/string on the Cupper project, whereas we had lots of pauses due to span tags. This change applies the same approach.